framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -7782,22 +7808,39 @@ VkDeviceAddress VulkanReplayConsumerBase::OverrideGetBufferDeviceAddress(
                                 "bufferDeviceAddressCaptureReplay feature for accurate capture and replay. The "
                                 "replay device does not support this feature, so replay may fail.");
     }
+    VkDevice                         device       = device_info->handle;
+    const VkBufferDeviceAddressInfo* address_info = pInfo->GetPointer();
+
+    // retrieve replay-time device-address
+    VkDeviceAddress replay_device_address = func(device, address_info);
 
     if (!device_info->allocator->SupportsOpaqueDeviceAddresses())
     {
+        // TODO: make this warning obsolete by re-mapping addresses (and a whole lot more ...)
```

Comment:
Should we make  an issue for this?  I'd like to have issues tracking TODOs that aren't going to be fixed in the near term (say 1-2 days) just so we don't loose track of this if you get redirected to something else in the near term.

Reply:
we do already track the progress for the overall initiative [here](https://github.com/LunarG/gfxreconstruct/issues/1526), but should make that clear in the TODO, under which condition/when we'll remove it again. will do!

Reply:
No need.  If it's tracked that's all.

Reply:
slightly improved wording for the TODO and added the issue-number, so they hopefully won't stay there for long :)